plugins/grpc/web/v2.0.0/buf.plugin.yaml

```diff
@@ -0,0 +1,24 @@
+version: v1
+name: buf.build/grpc/web
+plugin_version: v2.0.0
+source_url: https://github.com/grpc/grpc-web
+integration_guide_url: https://grpc.io/docs/platforms/web/quickstart
+description: gRPC for Web Clients.
+deps:
+  - plugin: buf.build/protocolbuffers/js:v3.21.4
```

Comment:
Didn't bump this to 4.0.0 yet - the upstream examples show 3.21.4: https://github.com/grpc/grpc-web/blob/875e2baadff1371a514d805c43a815e95355d499/net/grpc/gateway/examples/helloworld/package.json#L10